packages/flutter_tools/lib/src/project.dart

```diff
@@ -599,14 +599,11 @@ class AndroidProject extends FlutterProjectPlatform {
 Warning
 ──────────────────────────────────────────────────────────────────────────────
 Your Flutter application is created using an older version of the Android
-embedding. It is being deprecated in favor of Android embedding v2. Follow the
-steps at
+embedding. It is being deprecated in favor of Android embedding v2. Migrate
```

Comment:
Maybe "To migrate your project, follow the steps at" per [the recommendation against telling people to do something in the style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require). (That's in the documentation section, but seems relevant here.)